src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -248,6 +249,13 @@ private async Task PreRunConditions(WorkerRuntime workerRuntime)
             if (workerRuntime == WorkerRuntime.python)
             {
                 PythonHelpers.VerifyVirtualEnvironment();
+                // We need to update the PYTHONPATH to add worker's dependencies
+                var pythonPath = Environment.GetEnvironmentVariable("PYTHONPATH") ?? string.Empty;
+                var pythonWorkerDeps = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "workers", "python", "deps");
+                if (!pythonPath.Contains(pythonWorkerDeps))
+                {
+                    Environment.SetEnvironmentVariable("PYTHONPATH", $"{pythonWorkerDeps};{pythonPath}", EnvironmentVariableTarget.Process);
```

Comment:
use `Path.PathSeparator` instead of `;`. On Windows it's `;` but on Mac/Linux it's `:`